### ISSUE TITLE
Parser throws error on font shorthand

### ISSUE THREAD

**Issue by bartveneman** [shellscape/postcss-values-parser#63]
* Node Version: 10.0.0
* NPM Version: 6.5
* postcss-values-parser Version: 3.0.0-beta.2

This issue is regarding a problem with:
- [x] Standard CSS
- [ ] LESS
- [ ] SCSS
- [ ] SASS

If you have a large amount of code to share which demonstrates the problem you're experiencing, please provide a link to your
repository rather than pasting code. Otherwise, please paste relevant short snippets below.

```css
// offending or problematic css
.font-family3 { font: normal normal 1em/1 'Source Sans Pro', serif; }
```

```
// actual error output, if error was thrown
CssSyntaxError {
    column: 15,
    input: {
      column: 15,
      line: 1,
      source: 'normal normal 1em/1 \'Source Sans Pro\', serif',
    },
    line: 1,
    reason: 'Unknown word',
    source: 'normal normal 1em/1 \'Source Sans Pro\', serif',
    message: '<css input>:1:15: Unknown word',
  }
```

### Expected Behavior
Expected the font shorthand to be fully parsed in parts.

### Actual Behavior
It throws an error

### How can we reproduce the behavior?
```js
const {parse} = require('postcss-values-parser')
parse("normal normal 1em/1 'Source Sans Pro', serif")
```

**Comment by bartveneman** [comment]
If you're looking for more exotic font stuff, this is what I use to test projectwallace/css-analyzer: https://github.com/projectwallace/css-analyzer/blob/7dc6a0d7e093f408d5fd2e7db7da405fd5b2043d/test/analyzer/values/input.css#L145-L172

**Comment by shellscape** [comment]
Thanks for the issue. This is a weird one as it's not throwing an error for me - when I run that reproduction, I get a root node with the following in `root.nodes`:

```js
[ Word {
    raws: { before: '', after: '' },
    value: 'normal',
    type: 'word',
    isColor: false,
    isHex: false,
    isVariable: false,
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Word {
    raws: { before: ' ', after: '' },
    value: 'normal',
    type: 'word',
    isColor: false,
    isHex: false,
    isVariable: false,
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Numeric {
    raws: { before: ' ', after: '' },
    value: '1',
    type: 'numeric',
    unit: 'em',
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Operator {
    raws: { before: '', after: '' },
    value: '/',
    type: 'operator',
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Numeric {
    raws: { before: '', after: '' },
    value: '1',
    type: 'numeric',
    unit: '',
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Quoted {
    raws: { before: ' ', after: '' },
    value: '\'Source Sans Pro\'',
    type: 'quoted',
    quote: '\'',
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Punctuation {
    raws: { before: '', after: '' },
    value: ',',
    type: 'punctuation',
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } },
  Word {
    raws: { before: ' ', after: '' },
    value: 'serif',
    type: 'word',
    isColor: false,
    isHex: false,
    isVariable: false,
    parent:
     Root {
       raws: [Object],
       type: 'root',
       nodes: [Circular],
       source: [Object] },
    source: { start: [Object], input: [Input], end: [Object] } } ]
```

**Comment by bartveneman** [comment]
Ah, found a better way to reproduce:

```js
parse(`normal normal 1em/1 'Source Sans Pro', serif`).walk(node => {
	console.log(node)
})

// throw this.input.error('Unknown word', tokens[0][2], tokens[0][3]);
```

Perhaps I shouldn't be using the `.walk` method anymore?

**Comment by bartveneman** [comment]
I also found that it throws errors on the following values:

```css
grid-columns: 1/-1
font: 1em/ 1 serif
```
It does *not* throw when I add spacing around the `/`, so I'm starting to suspect that spacing around `/` could be at play here.

**Comment by shellscape** [comment]
OK something is going on in your environment. I've tried every example/repro you've posted here and every single one of them is working for me in a fresh environment. Would it help if I put up a small repo with them all to show them working?

I still suspect you have an old version of postcss-values-parser being loaded.

**Comment by bartveneman** [comment]
I cooked up a tiny repo for testing purposes: https://github.com/bartveneman/postcss-values-parser-v3.0.0-beta.2-testcase

**Comment by shellscape** [comment]
Thanks for doing that. I went through all of the test cases in index.js and all of them worked for me. The only difference between my setup and yours is I'm on Node 10.13.0. Could you have an old version of postcss-values-parser installed globally that is somehow taking precedence?

If you'd like to, we could setup a Google Hangout and do a screen share to try and figure this out.

**Comment by bartveneman** [comment]
I checked, but there is no global/other version for postcss-values-parser than the one in my package.json and package-lock.json. Is this error specific for v2?

**Comment by shellscape** [comment]
The last time I saw these types of errors, it was on v2. I'm really at a loss as to what could be causing errors on your end that don't appear on mine. I suggested the screen share so I could show that I wasn't making it up haha.

So I've managed to reproduce this, kind of. Something about parsing the font values on line 3 first, causes the `1/-1` value to fail. If I put any other parse in front of that, it works fine. So it's something about that value that's throwing things out of whack. I'll have to look into it further to see whats up. This is going to be a tough one to track down.

**Comment by shellscape** [comment]
Here's the culprit: https://github.com/shellscape/postcss-values-parser/blob/326151acf09cb1991d1bb5fba3ad041f3498f529/lib/nodes/Operator.js#L16

**Comment by shellscape** [comment]
Fixed in b6bcc5a. Thanks for working through this one with me.

**Comment by bartveneman** [comment]
Amazing, glad you were able to find it. Up to v3.0.0! 😁
